### PR TITLE
ADD some basis for automated integration regression testing

### PR DESCRIPTION
**Pull request by ndelangen**
Issue: -should create one-

## What I did
By adding examples we can both help users by giving them something to look at when starting their project; and also do regression tests by trying to run the examples with new version of storybook & addons.

## How to test
- not implemented yet -

### CODE REVIEW

**Comment by aaronmcadam** [comment]
This is a cool idea!

I'm not sure how Lerna deals with packages within packages though!

File: packages/examples/cra-storybook/.gitignore [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+# production
+/build
+
+# misc
+.DS_Store
+.env
+npm-debug.log*
+yarn-debug.log*
+yarn-error.log*
```
Comment (aaronmcadam) [inline packages/examples/cra-storybook/.gitignore]: I thought we were going to avoid Yarn until we Lerna's support is ironed out?

**Comment by aaronmcadam** [comment]
Does this PR solve the Lerna warning ("`Encountered a cycle in the dependency graph. This may cause instability!`")?

File: packages/examples/cra-storybook/.gitignore (hunk 1 above)
Comment (ndelangen) [inline packages/examples/cra-storybook/.gitignore]: This is generated >_<

**Comment by ndelangen** [comment]
> Does this PR solve the Lerna warning ("`Encountered a cycle in the dependency graph. This may cause instability!`")?

It doesn't

**Comment by ndelangen** [comment]
> I'm not sure how Lerna deals with packages within packages though!

I'm pretty sure it doesn't do anything to those, thus npm will install the version specified from the public repository. But I'm pretty sure we could find some ways..

**Pull request closed without merging**